src/components/SecondaryPanes/Frames/tests/FrameMenu.js

```diff
@@ -26,29 +29,63 @@ describe("FrameMenu", () => {
       stopPropagation: jest.fn(),
       preventDefault: jest.fn()
     };
+
+    L10N = {
+      getStr: jest.fn(value => value),
+      getFormatStr: jest.fn(value => value)
+    };
```

Comment:
We discussed this earlier, but i don't think you gain anything from stubbing this out. i.e. we dont really want to assert calls to it. and in general it will just work when it's not stubbed...